docs/src/Cli/GenerateCommand.php

```diff
@@ -48,7 +48,7 @@ final class GenerateCommand {
 	public function __invoke( $args, $assoc_args ) {
 		// Bail if current PHP version is not supported.
 		// Current combination of phpDocumentor and nikic/php-parser supports 7.4 or less.
-		if ( ! version_compare( phpversion(), '7.4', '<=' ) ) {
+		if ( version_compare( substr( phpversion(), 0, 3 ), '7.4', '>' ) ) {
```

Comment:
Why the `substr()` here?

```suggestion
		if ( version_compare( phpversion(), '7.4', '>' ) ) {
```

Reply:
version compare with is returning true in comparison due to patch version in it https://3v4l.org/H6Hnn#v7.4.33

Reply:
Ah, then probably better to just use `8.0`:

```suggestion
		if ( version_compare( phpversion(), '8.0', '>=' ) ) {
```

Reply:
Perfect ❤️